ROADMAP.md

```diff
@@ -2,14 +2,63 @@
-1. **More utils**. We want to work on new utility components: Spacing, Display, Color, etc.
-1. **Performance**. We can’t optimize something we can’t measure, and we don’t currently have a CI performance benchmark, so we will need to build one and start investigating bottlenecks.
-1. **Learning materials**. The quality of the documentation is equally as important as the quality of the implementation, and while the reference documentation is comprehensive, we could author a learning tutorial like Next.js has done, or an egghead.io course.
-1. **Themes**. We want to provide common layouts example to make getting started even easier. We also plan on adding more premium themes.
-1. **Documentation**. We want to translate the documentation into Chinese and other languages. Any help is welcomed!
-1. **Bundle size**. The library needs to be as small as possible, so we need to work on solutions to further reduce the bundle size, for example, supporting Preact or Nerv can help.
-1. **Type checking**. We will continue to improve the TypeScript and possibly the Flow coverage of the library.
-
-You can follow [our milestones](https://github.com/mui-org/material-ui/milestones) for a more detailed overview.
+## Methodology
+
+We work on the problems that resonate the most with our users.
+Please **upvote** 👍 the issues you are the most interested in on GitHub.
+Thank you for participating [in our developer survey](https://material-ui.com/blog/2019-developer-survey-results/).
+
+## Our priorities
+
+Here are our top priorities, ordered by importance:
+
+- 1.0 - **More components**. 🧰 People want more components. This is challenging for us to handle. Building and maintaining a professional component takes an incredible amount of time.
+Everybody underestimates it, we have a relatively small core team. We are going to try the following strategy:
+  - Identify frequently needed components. We have different leverages for this: the developer survey answers, the GitHub upvotes, the Algolia search volume, the Google search volume, documentation usage volume, npm downloads, etc.
+  - Prioritize the creation of frequently needed components.
+  - Optimize for the people maintaining the components. The more people maintaining the components the better.
+  - Encourage the usage of third-party components if they already exist and are well maintained. We can create [a search experience](https://xpvrpr7r4o.codesandbox.io/) for this.
+- 0.5 - **Better customization.** 💅 We want to make our component customization intuitive, no matter if you are using global CSS or styled-components:
+  - Improve the documentation.
+  - Better support for styled-components.
+  - Work on the core solution.
+  - Provide [more examples like this](https://mui-treasury.com/components/card).
```

Comment:
Might just be me, but I found the button page clearer as to what I was looking at. (i.e. customized Material-UI components.

Reply:
Sure, let's change it to the button page. I was hesitating.